### PR TITLE
Fix #1372  Layout Shift in Profile Page

### LINKED ISSUE #1372: Layout Shift in Profile Page
**Describe the bug**
In the profile screen of app, the layout shift can be seen, their is not symmetric spacing between different component. You can see the "Donate us" button in attached screenshot. Also there is issue of layout overflow.

**To Reproduce**
1. Go to `Profile` screen of app
2. Check the layout

**Expected behavior**
The spacing between elements should be symmetric. And there is should not be any layout shuft.

**Actual behavior**
The expected behaviour is discussed above. Currently the page is not responsive and layout shift happening.

**Screenshots**

### PR DESCRIPTION
**Pull request by Tanmoy741127**
**What kind of change does this PR introduce?**
There was a missing SizedBox between two components, so the gap between the components was too less. That has been fixed.

Alongside, the column was wrapped in a sizedbox [which size depends on screen height]. Which can give renderflex error any time. So that has been replaced with Outer SingleChildScrollView.

**Issue Number:**

Fixes #1372 

**Did you add tests for your changes?**

Not required as its an UI update

**Snapshots/Videos:**
| Before UI Update | After UI Update |
:-------------------------:|:-------------------------:
|  | 

**If relevant, did you update the documentation?**
Not Required

**Summary**

**Does this PR introduce a breaking change?**
Nope

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa/blob/master/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

**Review by palisadoes — CHANGES_REQUESTED** [review]
Please update the tests for the files you have updated. We need to get to 100% code coverage for them.

**Comment by palisadoes** [comment]
Please fix the failing tests

File: lib/views/after_auth_screens/profile/profile_page.dart [hunk 1]
```diff
@@ -94,154 +94,144 @@ class ProfilePage extends StatelessWidget {
...
+                            height: SizeConfig.screenHeight! * 0.05,
+                          ),
+                          CustomListTile(
+                            key: homeModel!.keySPAppSetting,
+                            index: 0,
+                            type: TileType.option,
+                            option: Options(
+                              icon: const Icon(
+                                Icons.phonelink_setup,
+                                size: 30,
```
Comment (SiddheshKukade) [inline lib/views/after_auth_screens/profile/profile_page.dart:110]: Wrap all the items with Flutter's `Expanded` Widget except the last one `from PALISADOES` to the bottom to ensure that it always sticks to the bottom regardless of the screen size.
More : https://api.flutter.dev/flutter/widgets/Expanded-class.html

**Review by SiddheshKukade — CHANGES_REQUESTED** [review]

**Comment by SiddheshKukade** [comment]
@Tanmoy741127 You mentioned https://github.com/PalisadoesFoundation/talawa/issues/1372#issuecomment-1350264244 changing the `My Tasks` sub-heading to occupy only single line is it also implemented ?

File: lib/views/after_auth_screens/profile/profile_page.dart (hunk 1 above)
Comment (JamaicanFriedChicken) [reply lib/views/after_auth_screens/profile/profile_page.dart:110]: @Tanmoy741127, Can you wrap the items with flutter's `Expanded` widget as Siddhesh recommended?

Comment (Tanmoy741127) [reply lib/views/after_auth_screens/profile/profile_page.dart:110]: @JamaicanFriedChicken sure. I have actually missed the comment. I will do that and update .

Comment (Tanmoy741127) [reply lib/views/after_auth_screens/profile/profile_page.dart]: @SiddheshKukade @JamaicanFriedChicken 
has wrapped the children in expanded in latest commit

**Comment by palisadoes** [comment]
@SiddheshKukade Do you have any additional comments?

**Comment by SiddheshKukade** [comment]
> @SiddheshKukade Do you have any additional comments?

@palisadoes no sir, changes seem reasonable to be merged now 👍🏻 .

**Review by SiddheshKukade — APPROVED** [review]

**Comment by palisadoes** [comment]
@Tanmoy741127 please update the tests for this file to get to 100% and work with@SiddheshKukade to meet this goal.

Thanks for finding this edge case

**Review by JamaicanFriedChicken — APPROVED** [review]

**Comment by Tanmoy741127** [comment]
> @Tanmoy741127 please update the tests for this file to get to 100% and work with@SiddheshKukade to meet this goal.
> 
> Thanks for finding this edge case

Ok will check.
